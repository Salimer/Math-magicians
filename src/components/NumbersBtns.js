function NumbersBtns() {
  return (
    <div className="numbers">
      <div className="button">AC</div>
      <div className="button">+/-</div>
      <div className="button">%</div>
      <div className="button">7</div>
      <div className="button">8</div>
      <div className="button">9</div>
      <div className="button">4</div>
      <div className="button">5</div>
      <div className="button">6</div>
      <div className="button">1</div>
      <div className="button">2</div>
      <div className="button">3</div>
      <div className="button">0</div>
      <div className="button">.</div>
    </div>
  );
}

export default NumbersBtns;
