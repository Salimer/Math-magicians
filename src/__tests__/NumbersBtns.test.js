import { render } from '@testing-library/react';
import NumbersBtns from '../components/NumbersBtns';

test('renders numbers buttons correctly', () => {
  const handleButtonClick = jest.fn();
  const { container } = render(
    <NumbersBtns handleButtonClick={handleButtonClick} />,
  );

  expect(container).toMatchSnapshot();
});
