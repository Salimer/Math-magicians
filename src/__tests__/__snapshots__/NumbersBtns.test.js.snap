// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders numbers buttons correctly 1`] = `
<div>
  <div
    class="numbers"
  >
    <button
      class="button"
      type="button"
    >
      AC
    </button>
    <button
      class="button"
      type="button"
    >
      +/-
    </button>
    <button
      class="button"
      type="button"
    >
      %
    </button>
    <button
      class="button"
      type="button"
    >
      7
    </button>
    <button
      class="button"
      type="button"
    >
      8
    </button>
    <button
      class="button"
      type="button"
    >
      9
    </button>
    <button
      class="button"
      type="button"
    >
      4
    </button>
    <button
      class="button"
      type="button"
    >
      5
    </button>
    <button
      class="button"
      type="button"
    >
      6
    </button>
    <button
      class="button"
      type="button"
    >
      1
    </button>
    <button
      class="button"
      type="button"
    >
      2
    </button>
    <button
      class="button"
      type="button"
    >
      3
    </button>
    <button
      class="button"
      type="button"
    >
      0
    </button>
    <button
      class="button"
      type="button"
    >
      .
    </button>
  </div>
</div>
`;
